pysolarfocus/components/base/part.py

```diff
@@ -0,0 +1,12 @@
+from abc import abstractmethod
+
+
+class Part(object):
+    """
+    Performing performance calculations
```

Comment:
Smart, but the comment is kind of misleading.
Maybe change it to something like "abstraction of a metric of the heating system".